### PR TITLE
Create; helper for table with various properties

### PR DESCRIPTION
**Pull request by chrisopedia**
Why is this pull request necessary:

~~❗ this is not mergeable as it relies on #70~~

1. adds in support for various versions of tables
2. includes documentation in mdcss form

Changes proposed in this pull request:

- small version
- bordered variant
- hover treatment
- striped version to give zebra striping
- responsive container

## Screenshot

image

Notify or mention any users: @jgallen23

### CODE REVIEW

File: conf/default-vars.yaml [hunk 1]
```diff
@@ -33,3 +33,13 @@ vars:
   heading-5-mobile-size: 14px
 
   stretch-spacing: 2px
+
+  table-cell-padding: .75rem
```
Comment (jgallen23) [inline conf/default-vars.yaml]: seems weird to start using rems now

File: conf/default-vars.yaml [hunk 2]
```diff
@@ -33,3 +33,13 @@ vars:
...
+  table-sm-cell-padding: .3rem
+  table-bg: transparent
+  table-bg-accent: '#f9f9f9'
```
Comment (jgallen23) [inline conf/default-vars.yaml]: colors should be in colors.yaml

Comment (jgallen23) [inline conf/default-vars.yaml]: unless we want to start moving colors to be in context with the vars.

@dawnerd thoughts?

**Comment by jgallen23** [comment]
can you post screenshot of docs?

**Comment by chrisopedia** [comment]
changed to pixels and added screenshot

**Pull request merged**